From 7e2766d52544286f5af9f6e0277443df4c2887a8 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Tue, 10 May 2011 13:44:51 +0800
Subject: [PATCH] ENGR00143179-1 esdhc: add usdhc_en flag into plat_data

The usdhc_en is used to control whether enable usdhc mode.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mmc.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mmc.h b/arch/arm/plat-mxc/include/mach/mmc.h
index c56268f..4cc69d5 100644
--- a/arch/arm/plat-mxc/include/mach/mmc.h
+++ b/arch/arm/plat-mxc/include/mach/mmc.h
@@ -43,6 +43,7 @@ struct mxc_mmc_platform_data {
 	unsigned int clk_always_on;     /* Needed by SDIO cards and etc */
 	unsigned int dll_override_en;	/* Enable dll override delay line */
 	unsigned int dll_delay_cells;	/* The number of delay cells (0-0x3f) */
+	unsigned int usdhc_en;
 	unsigned int reserved:16;
 	unsigned int card_fixed:1;
 	unsigned int card_inserted_state:1;
-- 
1.5.4.4

